figures/templates/figures/admin_dropdown_filter.html

```diff
@@ -0,0 +1,28 @@
+{# Attribution: Original code here: https://github.com/mrts/django-admin-list-filter-dropdown #}
+
+{% load i18n %}
+<script type="text/javascript">
+    var go_from_select = function(opt) { window.location = window.location.pathname + opt };
+</script>
+<h3>{% blocktrans with title as filter_title %} By {{ filter_title }} {% endblocktrans %}</h3>
+<ul class="admin-filter-{{ title|cut:' ' }}">
+{# Set the number after slice to the number of choices less one that trigger the selection list #}
+{% if choices|slice:"4:" %}
+    <li>
+    <select class="form-control" style="width: 95%;margin-left: 2%;"
+        onchange="go_from_select(this.options[this.selectedIndex].value)">
+    {% for choice in choices %}
+        <option{% if choice.selected %} selected="selected"{% endif %}
+         value="{{ choice.query_string|iriencode }}">{{ choice.display }}</option>
```

Comment:
`iriencode`! never heard about that before! Thanks for adding it.

Reply:
@OmarIthawi It's in the Django source too: https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/filter.html#L6

Reply:
Now I know, I just didn't know it existed :slightly_smiling_face: